petsitters/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db   import models
+
+from core.models import TimeStampModel
+
+class Petsitter(TimeStampModel):
+    name        = models.CharField(max_length=45)
+    title       = models.CharField(max_length=150)
+    price       = models.DecimalField(max_digits=8, decimal_places=2)
+    grade       = models.CharField(max_length=45)
+    information = models.TextField(max_length=2000)
+    address     = models.CharField(max_length=100)
+    longitude   = models.DecimalField(max_digits=9, decimal_places=6)
+    latitude    = models.DecimalField(max_digits=9, decimal_places=6)
+    types       = models.ManyToManyField('Type', through='PetsitterType')
+
+    class Meta:
+        db_table = 'petsitters'
+
+class PetsitterImage(models.Model):
+    petsitter_image_url = models.CharField(max_length=1000)
+    Petsitter_id        = models.ForeignKey('petsitter', on_delete=models.CASCADE)
```

Comment:
```suggestion
    petsitter_id        = models.ForeignKey('petsitter', on_delete=models.CASCADE)
```
대문자 P -> 소문자 p 로 수정해 주세요.